packages/broker/src/plugins/operator/OperatorFleetState.ts

```diff
@@ -103,6 +103,10 @@ export class OperatorFleetState extends EventEmitter<OperatorFleetStateEvents> {
         return [...this.heartbeatTimestamps.keys()]
     }
 
+    getPeerDescriptorOf(nodeId: NodeId): NetworkPeerDescriptor | undefined {
```

Comment:
```suggestion
    getPeerDescriptor(nodeId: NodeId): NetworkPeerDescriptor | undefined {
```

Reply:
It's not really a pure getter though since it requires an argument?